cypress/e2e/API/getPostman.cy.js

```diff
@@ -0,0 +1,47 @@
+/// <reference types="Cypress" />
+
+
+describe('Testando o endpoint /customers', () => {
+
+    it('Deve retornar uma lista de clientes com parâmetros de paginação e filtro', () => {
+      
+      const url = 'http://localhost:3001/customers?page=1&limit=10&size=Small';
+      
+      // Fazendo a requisição GET
+      cy.request({
+        method: 'GET',
+        url: url,
```

Comment:
Quando o nome da variável é o mesmo da propriedade, a variável pode ser omitida.

```suggestion
        url,
```